pymatgen/io/openmm/simulations.py

```diff
@@ -0,0 +1,100 @@
+"""
+This functionality eventually belongs in Atomate, but for the sake of rapidly developing
+a OpenMM IO, I will leave it here for now.
+
+The idea is that each function will operate on a openmm.Simulation and propagate the
+simulation forward in time.
+"""
+
+# base python
+from typing import Union, List
+
+# scipy
+import numpy as np
+
+# openmm
+from openmm import MonteCarloBarostat
+from openmm.unit import kelvin, atmosphere
+from openmm.app import Simulation
+
+
+__author__ = "Orion Cohen, Ryan Kingsbury"
+__version__ = "1.0"
+__maintainer__ = "Orion Cohen"
+__email__ = "orion@lbl.gov"
+__date__ = "Nov 2021"
+
+
+def equilibrate_pressure(
+    simulation: Simulation,
+    steps: int,
+    temperature: float = 298,
+    pressure: float = 1,
+):
+    """
+    Equilibrate the pressure of a simulation in the NPT ensemble.
+
+    Adds and then removes a openmm.MonteCarlo Barostat to shift the system
+    into the NPT ensemble.
+
+    Parameters
+    ----------
+    simulation : the openmm.Simulation to propagate.
+    steps : the length of the heating, holding, and cooling stages. Steps is number of steps.
+    temperature : temperature to equilibrate at (Kelvin)
+    pressure : pressure to equilibrate at (Atm).
+    """
+    context = simulation.context
+    system = context.getSystem()
+    assert (
+        system.usesPeriodicBoundaryConditions()
+    ), "system must use periodic boundary conditions for pressure equilibration."
+    barostat_force_index = system.addForce(MonteCarloBarostat(pressure * atmosphere, temperature * kelvin, 10))
+    context.reinitialize(preserveState=True)
+    simulation.step(steps)
+    system.removeForce(barostat_force_index)
+    context.reinitialize(preserveState=True)
+
+
+def anneal(
+    simulation: Simulation,
+    temperature: Union[float, int],
+    steps: List[int],
+    temp_steps: int = 100,
+):
+    """
+    Anneal the simulation at the specified temperature.
+
+    Annealing takes place in 3 stages, heating, holding, and cooling. The three
+    elements of steps specify the length of each stage.
+
+    Parameters
+    ----------
+    simulation : the openmm.Simulation to propagate.
+    temperature : the temperature reached for the holding stage (Kelvin)
```

Comment:
It looks like here you only specify the heating (holding) temperature, is that right? Is it assumed that the cooling goes back to 298K? Maybe make that explicit in the docstring, or add a kwarg for the starting/ending temperature.

Reply:
The system cools to whatever temperature it started at. I added to the docstring to make this clear.